content/javascript/concepts/setTimeout/setTimeout.md

```diff
@@ -0,0 +1,91 @@
+---
+Title: 'setTimeout'
+Description: setTimeout is a function serviced globally by the window object provided by the user's browser.
+Subjects:
+  - 'Web Development'
+  - 'Computer Science'
+Tags:
+  - 'Functions'
+  - 'Parameters'
+  - 'Arguments'
+CatalogContent:
+  - 'introduction-to-javascript'
+  - 'paths/front-end-engineer-career-path'
+---
+
+`setTimeout` is a function serviced globally by the window object provided by the user's browser, it is not part of JavaScript because it is not a JavaScript function.
```

Comment:
```suggestion
`setTimeout()` is a function serviced globally by the window object provided by the user's browser, it is not part of JavaScript because it is not a JavaScript function.
```

We'll refer to the function as such throughout the rest of the entry as well. JavaScript is available on the browser, Node.js is what we use to write in JavaScript locally